config/base/numaflow-server-sa.yaml

```diff
@@ -0,0 +1,4 @@
+apiVersion: v1
```

Comment:
Do we need a separate service account for numaflow-server? Currently I have re-used the existing one

Reply:
What is the best practice?